### ISSUE TITLE
FAILED: LeveldbAtLeastOnceDeliverySpec

### ISSUE THREAD

**Issue by drewhk** [akka/akka#20724]
https://jenkins.akka.io:8498/job/akka-nightly/3388/consoleFull

```
[info] - must re-deliver lost messages after restart *** FAILED *** (2 seconds, 989 milliseconds)
[info]   java.lang.AssertionError: assertion failed: expected Action(4,a-4), found Action(3,a-3)
[info]   at scala.Predef$.assert(Predef.scala:170)
[info]   at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:351)
[info]   at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:327)
[info]   at akka.testkit.TestKit.expectMsg(TestKit.scala:737)
[info]   at akka.persistence.AtLeastOnceDeliverySpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply$mcV$sp(AtLeastOnceDeliverySpec.scala:241)
[info]   at akka.persistence.AtLeastOnceDeliverySpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(AtLeastOnceDeliverySpec.scala:222)
[info]   at akka.persistence.AtLeastOnceDeliverySpec$$anonfun$1$$anonfun$apply$mcV$sp$2.apply(AtLeastOnceDeliverySpec.scala:222)
```

**Comment by johanandren** [comment]
Closed ticket #18864 seems to be about the same failure

**Comment by johanandren** [comment]
Tried with adding a time factor to increase timeouts specifically for the LevelDB variant, but didn't get that to work.
